mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -5662,4 +5666,85 @@ protected override StringBuilder AppendNamespace (StringBuilder buf, TypeReferen
 	}
 }
 
+internal abstract class FlagsFormatter {
+	public abstract string Format(long c, Dictionary<long, string> values, string typename);
```

Comment:
Space before `(`. This should likewise _probably_ be `IDictionary`, not `Dictionary`, and why `long`?